plugins/immer/README.md

```diff
@@ -0,0 +1,45 @@
+# Rematch Immer
+
+Immer plugin for Rematch. Provides immutable ability on immer library.
+
+### Install
+
+```
+npm install @rematch/immer
+```
+
+### Setup
+
+```js
+import selectorsPlugin from '@rematch/immer'
+import { init } from '@rematch/core'
+
+const immer = immerPlugin()
+
+init({
+  plugins: [immer]
+})
+```
+
+### Usage
+
+Reducer could be use mutable method to achieve immutable state, but this plugin may be some different that reducers must return value because literal don't support immer.
```

Comment:
Does this mean that a state cannot have a primitive value?

Reply:
No, state can use primitive value, but must return changed value. because immer use observable to detect object or array change state. That means reducers don't return any change value, immer will be detected the change.

In there document, I just want to unify the reducer method.

Reply:
So Immer only supports change detection on plain objects and arrays, but primitive values like strings or numbers will always return a change?

Reply:
Yes, that is observable limited, only to detect object or array.

Reply:
Definitely worth mentioning then. Could we rephrase it in the README to say:

```
In Immer, reducers perform mutations to achieve the next immutable state. Keep in mind, Immer only supports change detection on plain objects and arrays, so primitive values like strings or numbers will always return a change.
``

Reply:
good, I will modified the document.